main/src/main/scala/org/clulab/numeric/mentions/DateRangeMention.scala

```diff
@@ -19,8 +19,8 @@ class DateRangeMention ( labels: Seq[String],
   // does not have a year value but date2Norm has a year value
   // This will handle date ranges of form month date and month date of year.
   // Ex: "Sowing dates between May 3 and October 2, 2020" will have a normalized value of 2020-05-03 -- 2020-10-02
-  if (!date1Norm.equals("XXXX-XX-XX") && date1Norm.contains("XXXX")) {
-    if (!(date2Norm contains "XXXX")) {
+  if (!date1Norm.equals("XXXX-XX-XX") && date1Norm.startsWith("XXXX")) {
+    if (!(date2Norm.equals("ref-date") || date2Norm.startsWith("XXXX"))) {
```

Comment:
Has someone been writing too much Java ;-) ?  You almost fooled me.  Is this `date1Norm != "XXXX-XX-XX"`?

Reply:
😅 yes, you a right. I'll fix it right now!
By the way, looking at these 2 lines, the DateNorm companion object you were proposing makes even more sense.